dockers/docker-database/supervisord.conf.j2

```diff
@@ -39,12 +39,10 @@ dependent_startup=true
 command=/bin/bash -c "{ [[ -s /var/lib/{{ redis_inst }}/dump.rdb ]] || rm -f /var/lib/{{ redis_inst }}/dump.rdb; } && mkdir -p /var/lib/{{ redis_inst }} && exec /usr/bin/redis-server /etc/redis/redis.conf --bind {{ LOOPBACK_IP }} {{ redis_items['hostname'] }} --port {{ redis_items['port'] }} --unixsocket {{ redis_items['unix_socket_path'] }} --pidfile /var/run/redis/{{ redis_inst }}.pid --dir /var/lib/{{ redis_inst }}"
 priority=2
 user=redis
-autostart=false
+autostart=true
 autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
-dependent_startup=true
```

Comment:
Does `dependent_startup` need to be removed for the other entries here? Or is it just Redis that it needs to be removed for?

Reply:
just remove redis. Other processes still need dependent_startup, for example, containercfgd should start after rsyslogd.